inc/saischedulergroup.h

```diff
@@ -55,6 +52,10 @@ typedef enum _sai_scheduler_group_attr_t
        MANDATORY_ON_CREATE,  CREATE_ONLY */
     SAI_SCHEDULER_GROUP_ATTR_LEVEL = 0x00000004,
 
+    /** Maximum Number of childs on group [uint8_t]
+     * MANDATORY_ON_CREATE,  CREATE_ONLY */
+    SAI_SCHEDULER_GROUP_ATTR_MAX_CHILDS = 0x00000000,
```

Comment:
Intention is app to set the maximum child links as part of the SG creation. NPU has global node limits might be verified through error once limits exceeds.